gradle.properties

```diff
@@ -14,15 +14,14 @@ springDependencyManagementPluginVersion=1.0.3.RELEASE
 
 #         springBootVersion = '1.5.6.RELEASE'
 
-eventuateClientVersion=0.22.0.RC2
-eventuateLocalVersion=0.31.0.RC5
-
-eventuateTramVersion=0.22.0.RC7
-eventuateTramSagasVersion=0.12.0.RC6
-
-eventuateUtilVersion=0.2.0.RELEASE
-eventuateCommonVersion=0.8.3.RELEASE
-eventuateMessagingKafkaVersion=0.5.0.RELEASE
+eventuateClientVersion=0.22.0-SNAPSHOT
```

Comment:
These can be replaced with replaced versions.

Reply:
will do

Reply:
replaced, except kafka and sagas, it seems, they are not released.